Source/Scene/PointCloudEyeDomeLighting.js

```diff
@@ -210,39 +209,24 @@ define([
         if (!defined(shader)) {
             var attributeLocations = shaderProgram._attributeLocations;
 
-            var vs = shaderProgram.vertexShaderSource.clone();
             var fs = shaderProgram.fragmentShaderSource.clone();
 
-            vs.sources = vs.sources.map(function(source) {
-                source = ShaderSource.replaceMain(source, 'czm_point_cloud_post_process_main');
-                return source;
-            });
-
             fs.sources = fs.sources.map(function(source) {
                 source = ShaderSource.replaceMain(source, 'czm_point_cloud_post_process_main');
                 source = source.replace(/gl_FragColor/g, 'gl_FragData[0]');
                 return source;
             });
 
-            vs.sources.push(
-                'varying vec3 v_positionEC; \n' +
-                'void main() \n' +
-                '{ \n' +
-                '    czm_point_cloud_post_process_main(); \n' +
-                '    v_positionEC = (czm_inverseProjection * gl_Position).xyz; \n' +
-                '}');
             fs.sources.unshift('#extension GL_EXT_draw_buffers : enable \n');
             fs.sources.push(
-                'varying vec3 v_positionEC; \n' +
                 'void main() \n' +
                 '{ \n' +
                 '    czm_point_cloud_post_process_main(); \n' +
-                // Write log base 2 depth to alpha for EDL
-                '    gl_FragData[1] = vec4(v_positionEC, log2(-v_positionEC.z)); \n' +
+                '    gl_FragData[1] = czm_packDepth(gl_FragCoord.z); \n' +
```

Comment:
I wanted to still use a varying instead of `gl_FragCoord.z`, since apparently on some platforms this has the same precision problems as depth textures, but I couldn't get it to work for some reason... the EDL effect happened, but it seemed like the depth value was wrong in some places.

However! I've only observed the precision problem on newer iPads, and the iPad lacks other necessary extensions, so in practice it shouldn't make a difference to just use `gl_FragCoord.z`. It's also cleaner.

Reply:
For posterity: https://github.com/mrdoob/three.js/issues/9092#issuecomment-367436360